### PR TITLE
Hazardous Waste and Used Oil Facilities

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the [Utah DWMRC Hazardous Waste and Used Oil Facilities](https://gis.utah.gov/products/sgid/environment/hazardous-waste-facilities/). Let me know what you think. Thank you!

### CODE REVIEW

File: metadata/environment/dwmrc_hazardous_waste_and_used_oil_facilities.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+This dataset is used in many waste disposal information lookups, recycling maps, and other resources. These data points have been made available to the public for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Each point location in this dataset represents a hazardous waste or used oil facility. Features contain the facility name, street address, facility number, and type.
+
+### How was the dataset created?
+
+The Divison of Waste Management and Radiation Control (DWMRC) derived this dataset from information submitted by individual waste management facilities. This dataset nightly through an automated script with new facilities and information updates.
```
Comment (steveoh) [inline metadata/environment/dwmrc_hazardous_waste_and_used_oil_facilities.md:33]: > This dataset nightly through an automated script with new facilities and information updates.

This sentence reads funny.

File: metadata/environment/dwmrc_hazardous_waste_and_used_oil_facilities.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Data Page Link
+
+[https://gis.utah.gov/products/sgid/environment/hazardous-waste-facilities/]
+
+## Update
+
+### Update Schedule
+
+This dataset is updated on a nightly basis.
```
Comment (steveoh) [inline metadata/environment/dwmrc_hazardous_waste_and_used_oil_facilities.md:71]: **question**: is this true? what tool is handling that?

**Comment by stdavis** [comment]
FYI - https://github.com/agrc/porter/issues/360

**Comment by miriamseely** [comment]
Looks like this one is being voted off the island, so I'll go ahead and close this request. Thank you @stdavis @steveoh!

**Pull request closed without merging**